### DISCUSSION TITLE
How does pinia work with Reactivity Tranform?

### DISCUSSION THREAD

**Discussion by martinszeltins** [vuejs/pinia#1333] [Help and Questions]
For example if I get this value from my store, it is actually wrapped in `Ref` and so in order to use it I need to add `.value` but is there a way to use Vue's new Reactivity Tranform so I would not have to write `.value`?

```js
const { isSettingsLoaded } = storeToRefs(settingsStore)

// isSettingsLoaded.value
```

**Comment by posva** [comment]
You need to use `$(storeToRefs(settingsStore))`: https://vuejs.org/guide/extras/reactivity-transform.html#destructuring-with
